commons/ava_services/gecko_service_initializer_core.go

```diff
@@ -109,11 +114,12 @@ func (core GeckoServiceInitializerCore) InitializeMountedFiles(osFiles map[strin
 func (core GeckoServiceInitializerCore) GetStartCommand(mountedFileFilepaths map[string]string, publicIpAddr string, dependencies []services.Service) ([]string, error) {
 	numBootNodeIds := len(core.bootstrapperNodeIds)
 	numDependencies := len(dependencies)
-	if numDependencies != numBootNodeIds {
+	if numDependencies > numBootNodeIds {
```

Comment:
J/w, what's the situation where you'd want to specify less boot nodes than exist?

Reply:
This should actually be done by starting one node with no boot nodes and then launching the rest with the original node to bootstrap off of. This is because the node that is started with no boot nodes, will start immediately. However, if a node is not done bootstrapping then it will drop bootstrapping messages received from its peers. So when we're starting a blank network with no actual data to bootstrap, we should ideally start one node and the rest off of only that node.